### DISCUSSION TITLE
Anzeige des Datums ändern

### DISCUSSION THREAD

**Discussion by jo3rn** [jo3rn/blockchAIn#10] [Q&A] [answered]
Die [Dokumentation zum DateTimeFormatter](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/time/format/DateTimeFormatter.html) gibt Aufschluss darüber, warum das in der `Block`-Klasse verwendete Muster ("uuuu.MM.dd-HH:mm:ss") diese Art von `timestamp` erzeugt: `2021.02.20-11:00:00`.

Wie muss das Muster erweitert werden, um zusätzlich noch den Tag des Jahres anzuzeigen, sodass folgendes ausgegeben wird:
`2021.03.24-14:00:00 (der 83. Tag des Jahres)`

Tipp: escape

**Comment by ActionHOSchT** [comment, marked as answer]
"uuuu.MM.dd-HH:mm:ss'(der 'D'. Tag des Jahres)'"

![grafik](https://user-images.githubusercontent.com/73703991/113851853-384a0700-979c-11eb-9392-d6db4340ba2d.png)

![grafik](https://user-images.githubusercontent.com/73703991/113851958-4dbf3100-979c-11eb-9429-62d0e2b33a1d.png)

**Reply by jo3rn** [reply]
siehe auch: #14
